deployment/terraform/assets/provisioners/metrics.sh

```diff
@@ -41,6 +41,12 @@ do
       sudo mkdir /opt/yace && \
       wget https://github.com/nerdswords/yet-another-cloudwatch-exporter/releases/download/v0.61.2/yet-another-cloudwatch-exporter_0.61.2_Linux_x86_64.tar.gz && \
       sudo tar -zxf yet-another-cloudwatch-exporter_0.61.2_Linux_x86_64.tar.gz -C /opt/yace && \
+      # Install Loki
+      mkdir -p /etc/apt/keyrings/ && \
+      sudo bash -c 'wget -q -O - https://apt.grafana.com/gpg.key | gpg --dearmor > /etc/apt/keyrings/grafana.gpg' && \
+      sudo bash -c 'echo "deb [signed-by=/etc/apt/keyrings/grafana.gpg] https://apt.grafana.com stable main" | tee /etc/apt/sources.list.d/grafana.list' && \
+      sudo apt-get -y update && \
+      sudo apt-get install -y loki && \
```

Comment:
Wondering if it would be best to follow what we did for other Grafana packages and install from the `.deb` so we can pin the version and avoid surprises.

Reply:
Good point, will take a look